src/Quarrel.ViewModels/Extensions/System.Collections.Generic/HashedCollection.cs

```diff
@@ -125,7 +125,7 @@ public bool ContainsKey(TKey key)
 
         public bool ContainsValue(TValue value)
         {
-            return _dict.ContainsValue(value);
+            return _dict.Values.Any(v => v.Equals(value));
         }
```

Comment:
What is the purpose of this change?

Reply:
ConcurrentDictionary<,> doesn't have ContainsValue.  The code is functionally equivalent as both must iterate and compare each Value until a match is found.